arm-servermanagement/2016-07-01-preview/servermanagement.json

```diff
@@ -1580,6 +1581,14 @@
               "type": "string",
               "format": "date-time",
               "description": "the date/time of the last published gateway"
+            },
+            "installerDownload": {
```

Comment:
@amarzavery, since the API version indicates that this is a preview, my assumption is that we are accepting breaking changes. Especially if we mark the installerDownload as readOnly (since it is, correct?)